internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -339,10 +357,41 @@ func (cf CloudFormation) executeAndRenderChangeSet(in *executeAndRenderChangeSet
 	if err != nil {
 		return err
 	}
+	var sigChannel chan os.Signal
+	if in.enableInterrupt {
+		sigChannel = cf.notifySignals()
+	}
+	g, ctx := errgroup.WithContext(context.Background())
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+	g.Go(func() error {
+		defer cancel()
+		if err := cf.renderChangeSet(ctx, changeSetID, in); err != nil {
+			if !in.enableInterrupt {
+				return err
+			}
+			if !errors.Is(err, context.Canceled) {
+				return err
+			}
```

Comment:
i think this can be simplified to just
```suggestion
			if !errors.Is(err, context.Canceled) {
				return err
			}
```
since the error will never be `context.Canceled` if `in.enableInterrupt` is false

Reply:
sorry 😞 I missed this